files/chrono/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "chrono",
+  "repo": "wanasit/chrono",
+  "files": {
+    "include": ["chrono.js", "chrono.min.js", "chrono.map.json"]
```

Comment:
There is no map.json here https://github.com/wanasit/chrono/tree/v1.0.2
Also I think that just `chrono.min.js` is enough